components/ui/header.tsx

```diff
@@ -36,6 +36,9 @@ export default function Header() {
           <nav className="hidden md:flex md:grow">
             {/* Desktop sign in links */}
             <ul className="flex grow justify-end flex-wrap items-center">
+              <li>
+                <a href="/pricing" className="font-medium text-gray-1000 hover:text-purple px-5 py-3 flex items-center transition duration-150 ease-in-out mr-2">Pricing</a>
+              </li>
               <li>
                 <a href="https://app.onestudy.io/sign-in" className="font-medium text-gray-1000 hover:text-purple px-5 py-3 flex items-center transition duration-150 ease-in-out">Sign in</a>
```

Comment:
i guess this wasn't a change in this PR but wdyt about giving this button a purple outline?